arch/arm64/boot/dts/rockchip/rk3588-blueberry-r58-4X4-hugsun.dtsi

```diff
@@ -0,0 +1,326 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+/*
+ * Copyright (c) 2021 Rockchip Electronics Co., Ltd.
+ *
+ */
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/pwm/pwm.h>
+#include <dt-bindings/pinctrl/rockchip.h>
+#include <dt-bindings/input/rk-input.h>
+#include <dt-bindings/display/drm_mipi_dsi.h>
+#include <dt-bindings/display/rockchip_vop.h>
+#include <dt-bindings/sensor-dev.h>
+#include "rk3588-cpu-swap.dtsi"
+
```

Comment:
Has anyone seen this stuff before? Seems it "swaps" the big cores with the little ones. 
How does this make any sense....?